### DISCUSSION TITLE
Initial test availability

### DISCUSSION THREAD

**Discussion by pankaj87072** [ruxailab/RUXAILAB#369] [Q&A]
when starting local environment contributor should get initial test right.having some initial tests available for contributors can significantly reduce the time it takes to get started with contributing to a project. These tests serve as a baseline for understanding the expected behavior of the codebase and help contributors ensure that their changes do not introduce regressions or break existing functionality.

**Comment by pankaj87072** [comment]
hey @jvJUCA @KarinePistili any comments??

**Reply by KarinePistili** [reply]
Hello @pankaj87072 , it is a nice suggestion, maybe it could be created an optional script to upload this example data. What information do you think would be nice to have? If you feel like we can elaborate a little more about a solution and create an issue for this.
